src/main/java/com/robinpowered/sdk/model/Organization.java

```diff
@@ -17,11 +17,21 @@
 
 
     /**
+     * Properties
+     */
+
+    private UserAccess userAccess;
+
+
+    /**
      * Methods
      */
 
-    public Organization(int id, String slug, boolean isOrganization, DateTime createdAt, DateTime updatedAt) {
+    public Organization(int id, String slug, boolean isOrganization, DateTime createdAt, DateTime updatedAt,
+                        UserAccess userAccess) {
```

Comment:
This shouldn't be provided in the constructor. We only require final attributes in our model constructors, and the others are set by getters/setters. Otherwise this model is always required to construct an org object, which makes no sense. 

There's another concern here, though, which is the the `UserAccess` property of an or is user-specific. Really any truly when getting `/users/:id/organizations` you're getting back "User Org" vs. an "Org"... that might seem confusing. Lemme play around with this branch and see what might be a good fit, cause this isn't an easy design to figure out.